cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -183,16 +181,16 @@
 {% from 'molecules/notification.html' import render as notification with context %}
 {% import 'organisms/post-preview.html' as post_preview with context %}
 
-{% macro render(controls, index) %}
+{% macro render(controls) %}
     <div class="o-filterable-list-controls"
-         id="o-filterable-list-controls-{{ index | string }}">
+         id="o-filterable-list-controls">
         {% set form = filter_data.forms.pop(0) %}
         {% set posts = filter_data.page_sets.pop(0) %}
-        {% set has_active_filters = page.has_active_filters(request, index) %}
+        {% set has_active_filters = request.GET %}
```

Comment:
is it possible that there might be query string params not associated with the filters ?

Reply:
Agree, this feels like an oversimplification. It's likely there aren't usually other query string parameters but this is not a safe assumption to make.

Reply:
This is fixed in https://github.com/cfpb/cfgov-refresh/pull/3628/commits/1bc68c6ea7f779bf6c373d888c24eeebc2f78df6